implementations/rust/ockam/ockam_command/src/logs/mod.rs

```diff
@@ -64,14 +64,13 @@ pub fn setup_logging(
         }
         // If a log path is provided, log to a rolling file appender.
         Some(log_path) => {
-            let r = RollingFileAppender::new(
-                log_path,
-                RollingConditionBasic::new()
-                    .daily()
-                    .max_size(log_max_size_bytes()),
-                log_max_files(),
-            )
-            .expect("Failed to create rolling file appender");
+            let log_dir = log_path.parent().expect("Failed to get log directory");
```

Comment:
Can we easily refactor the logging logic between the app and the command line?

Reply:
Probably, I'll give it a try

Reply:
Done!